web/core/components/cycles/analytics-sidebar/issue-progress.tsx

```diff
@@ -17,9 +17,8 @@ import { EIssueFilterType, EIssuesStoreType } from "@/constants/issue";
 import { getDate } from "@/helpers/date-time.helper";
 // hooks
 import { useIssues, useCycle, useProjectEstimates } from "@/hooks/store";
-// plane web constants
-import { SidebarBaseChart } from "@/plane-web/components/cycles/analytics-sidebar";
 import { EEstimateSystem } from "@/plane-web/constants/estimates";
+import SidebarChart from "./sidebar-chart";
```

Comment:
Previously it was coming from Plane-web. Is this intentional and wont it cause any issues in EE?

Reply:
Resolved